apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerServlet.java

```diff
@@ -202,18 +202,13 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
 
     private void processException(PeerImpl peer, Exception e) {
         if (peer != null) {
-//            if ((Peers.communicationLoggingMask & Peers.LOGGING_MASK_EXCEPTIONS) != 0) {
-//                if (e instanceof RuntimeException) {
-//                    LOG.debug("Error sending response to peer " + peer.getHost(), e);
-//                } else {
-//                    LOG.debug(String.format("Error sending response to peer %s: %s",
-//                        peer.getHost(), e.getMessage() != null ? e.getMessage() : e.toString()));
-//                }
-//            }
-             LOG.debug("Error sending response to peer " + peer.getHost(), e);
+
 //jetty misused this, ignore            
             if(!(e instanceof ClosedChannelException )){
+                LOG.debug("Error sending response to peer " + peer.getHost(), e);
                 peer.blacklist(e);
+            }else{
```

Comment:
Do we really need to blacklist client, who's channel was closed ?
`peer.blacklist(e);`